0004-ecs2ext-protocol/README.md

```diff
@@ -49,65 +50,51 @@ Example of such case is push notifications from server to client.
 Endpoint and extension can send and receive messages with `payload`.
 
 
-### Formats
-#### Avro JSON
+## Message exchange
 ![](ecs2ext-ipc.png?raw=true)
-##### Message from ECS to extension
-ECS should send message using NATS to the next subject:
-```
-kaa.{protocol-version}.service.{service-instance-id}
-```
-Also, ECS should include NATS `replyTo` field which should point to ECS replica that will handle response:
-```
-kaa.{protocol-version}.service.{service-instance-id}.{service-instance-replica-id} 
-```
-Refer to [Kaa event-based protocol documentation](/) for `protocol-version`, `service-instance-id` and `service-instance-replica-id` parameters.
 
-Format of the ECS-to-extension message:
-- `correlationId` (string, required) - refer to [Kaa event-based protocol documentation](/) for description.
-- `timestamp` (number, required) - message creation timestamp.
-- `timeout` (number, required) - amount of milliseconds from `timestamp`. Value '0' means "the request does not time out".
-- `appVersionToken` (string, required) - refer to [??](/) for description.
+#### Message from ECS to extension
+ECS should send message using NATS to the next subject: `kaa.v1.service.{service-instance-name}`.
```

Comment:
This is not a good topic to send all message types to. Should we add `ecs2ext` suffix at the end of the subject?
@vtkhir @Deepnekroz